apps/web/src/utils/md.ts

```diff
@@ -19,6 +19,22 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import snarkdown from "snarkdown";
 
+function addAttributes(
+  html: string,
+  tag: keyof HTMLElementTagNameMap,
+  attributes: Record<string, string>
+) {
+  const temp = document.createElement("div");
+  temp.innerHTML = html;
+  const elements = temp.querySelectorAll(tag);
+  elements.forEach((element) => {
+    Object.entries(attributes).forEach(([key, value]) => {
+      element.setAttribute(key, value);
+    });
+  });
+  return temp.innerHTML;
+}
```

Comment:
What is this for?

Reply:
`snarkdown` wasn't adding `target=_blank` to `a` tags, so I'm using this function to achieve that